discord/embeds.py

```diff
@@ -228,9 +228,15 @@ def from_dict(cls, data: Mapping[str, Any]) -> Self:
         # try to fill in the more rich fields
 
         try:
-            self._colour = Colour(value=data['color'])
+            color = data['color']
         except KeyError:
-            pass
+            try:
+                color = data['colour']
+            except KeyError:
+                color = None
+
+        if color is not None:
+            self._colour = Colour(value=color)
```

Comment:
What happens if color is `None`? Is `_colour` referenced anywhere else where it might break?

Reply:
See the `colour` property starting on line 318, as well as the fact that `self._colour` would also not be set on this logical path in current master